charts/hail-search/templates/_helpers.tpl

```diff
@@ -46,6 +34,6 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 Selector labels
 */}}
 {{- define "hail-search.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "hail-search.name" . }}
+app.kubernetes.io/name: {{ include "hail-search.fullname" . }}
```

Comment:
I would proceed with caution when updating selectorLabels -- I believe that it is an immutable field on the k8s API, and changing what goes in here will require deleting the running deployment, and redeploying.

I don't know how many folks out there are actually using these helm charts besides us, so we may no tneed to worry too much, but figured I'd mention it.

Reply:
👍 good call out.  I think it'll just impact us at the moment, but this is definitely a change for us from `hail-search` to `seqr-hail-search` (including the "release name").